data/sql/updates/pending_db_world/rev_1592648244203077800.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1592648244203077800');
+
+ALTER TABLE `player_factionchange_titles`
+	ADD `alliance_comment` TEXT AFTER `alliance_id`,
+	ADD `horde_comment` TEXT AFTER `horde_id`;
+	
+REPLACE INTO `player_factionchange_titles` (`alliance_id`, `alliance_comment`, `horde_id`, `horde_comment`) VALUES
```

Comment:
prefere `DELETE/INSERT` over `REPLACE`

Reply:
Me too, but the only reason I did this is because I don't want to do 
```sql
DELETE `player_factionchange_titles` 
```
(Since this affects the whole table I chose this over other since it would take too much time to DELETE alliance_id IN (X, Y) AND `horde_id` IN (Z, D)